homeassistant/components/conversation/default_agent.py

```diff
@@ -197,6 +194,28 @@ async def async_process(self, user_input: ConversationInput) -> ConversationResu
             response=intent_response, conversation_id=conversation_id
         )
 
+    def _recognize(
+        self, user_input: ConversationInput, lang_intents: LanguageIntents
+    ) -> RecognizeResult | None:
+        """Search intents for a match to user input."""
+        slot_lists: dict[str, SlotList] = {
+            "area": self._make_areas_list(),
+            "name": self._make_names_list(),
```

Comment:
These are async methods which you're now calling from sync context. Should be moved back.